scripts/kind-cluster.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+set -eux
+
+DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+KIND_IMAGE="kindest/node:v1.27.3"
+
+# deploy_kind installs the kind cluster
+deploy_kind() {
+  cat <<EOF | kind create cluster --image ${KIND_IMAGE} --config=- --kubeconfig=${DIR}/kubeconfig
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+networking:
+    podSubnet: $NET_CIDR_IPV4,$NET_CIDR_IPV6
+    serviceSubnet: $SVC_CIDR_IPV4,$SVC_CIDR_IPV6
+    ipFamily: $IP_FAMILY
+nodes:
+- role: control-plane
+  kubeadmConfigPatches:
+  - |
+    kind: ClusterConfiguration
+    apiServer:
+        extraArgs:
+            v: "5"
+    controllerManager:
+        extraArgs:
+            v: "5"
+    scheduler:
+        extraArgs:
+            v: "5"
+- role: worker
+- role: worker
+EOF
+}
+
+# install_netobserv-agent will install the daemonset
+# into each kind docker container
+install_netobserv-agent() {
+docker build . -t localhost/ebpf-agent:test
+kind load docker-image localhost/ebpf-agent:test
+kubectl apply -f ${DIR}/agent.yml
+}
```

Comment:
```suggestion
install_netobserv-agent() {
  docker build . -t localhost/ebpf-agent:test
  kind load docker-image localhost/ebpf-agent:test
  kubectl apply -f ${DIR}/agent.yml
}
```

Reply:
is it usual to name `localhost` an image ?
we could use $USER here

Reply:
do u mean quay with $USER ? if so that is very slow during docker image pull the above is much faster

Reply:
Just using `$USER` value to name the image. No need to push to quay.
That's what we do in MAKEFILE:
```
# In CI, to be replaced by `netobserv`
IMAGE_ORG ?= $(USER)
```

Reply:
ok sure, good to know ! 
Keep it as is then :smile_cat: